number_of_pods_in_system.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# This script checks the number of pods in a given namespace
+# It waits untill we have the minimum number of pods to start working on the system
+# 
+# It was created simnce after nb install sometimes the backingstore is not created yet
+# In k8s we cannot wait for a resource that doesn't exist
```

Comment:
I think we can remove it.
or rephrase completely.

Reply:
I tried to add information, please tell me what is stil not clear.